### ISSUE TITLE
JibberJabber

### ISSUE THREAD

**Issue by jewbmx** [jewbmx/repo#5]
Fair warning to anyone that uses my github repos for any reason, imma be doing a deep clean soon and probably ditching some repos while im at it. So take what ya want and/or make the necessary changes.

Will also eventually be uploading a new version of scrubs once i finish all the stuff im doing to it. Its gonna be for kodi 19 maybe workin in older kodi versions, will also be english only and will no longer have debrid bullshit in it either. A whole lotta other changes as well but who cares lol.

**Comment by AT5Pin** [comment]
So happy to see you post up in here again. missed your excellent work you gave to us with V2. look forward to any future work you do. simply the best! Thank you, for doing your do!

**Comment by Lord-Ariak** [comment]
Yoh! ma'man!! Yo Repo is worth the wait.

**Comment by jewbmx** [comment]
Did yall know our trakt profiles are apparently public? Thought my episode addiction was private knowledge lol https://trakt.tv/users/jewbmx

**Comment by AT5Pin** [comment]
No, didn't know that, but I'm not surprised. Gonna have to pull my tin foil hat down a little tighter...

**Comment by bigpoppapaul** [comment]
> Did yall know our trakt profiles are apparently public? Thought my episode addiction was private knowledge lol https://trakt.tv/users/jewbmx

You can set it to private if you want. 
![Screenshot_20210817-154228_Chrome](https://user-images.githubusercontent.com/20094991/129790191-ca1b833f-ea4c-45f8-92ab-8268019d91bd.png)

**Comment by AT5Pin** [comment]
Thanks bigpoppapaul. I haven't set up trakt yet with my latest setup. I'm a simple user and the basic configuration works for me. no Trakt, no skins, just a couple of addons is all my old azz, feeble minded self can handle, but I'll make it private when I get around to it. I do hope jewbmx's pal Tempest comes back tho. these two guys is all I needed.

**Comment by jewbmx** [comment]
I didnt know about the private option thats a nice choice lol, although i might leave my account public that way if i ever happen to forget my login info i can just scrape my stuff easy by viewing my own page lol

**Comment by jewbmx** [comment]
Im doing some major remodeling in the indexers and at the moment doing tmdb shit.
Im tellin ya this because id like to do a little ghetto poll and ask you all something, plus it gives you all a chance to toss in ideas or requests and yada.yada..yada...

So my poll or question is if i were to add in tmdb user / account features ( like trakt is now. ) with their watch list and all that stuff,
Who would actually use it and want it? lol

Personally i think it would be handy since i use a tmdb app on my phone and one for trakt, plus the imdb ones already in the addon even tho i dont like it or use the feature lmao.

**Comment by jewbmx** [comment]
if anyones curious on whats taking me so long or what i been doing lately or what hell is like, you can check these tmdb shit piles out lmao... https://github.com/jewbmx/repo/tree/master/ftp
was thinking the collections list was bad but these 3 are wild lol. stupid ocd wont let me skip em and move on either.

**Comment by jewbmx** [comment]
Anyone else just love deleting the __pycache__ folders instead of having to fish out and delete them dreadfully awful .pyo files?

**Comment by jewbmx** [comment]
If anyone is interested in using my tmdb module i been making lately, i just finished uploading it to the ScraperWork folder. Its done in my eyes and fully tested on my end so now its "released as a beta version" lol.

**Comment by jewbmx** [comment]
Think i broke the .io bit of the repo for installation of the repo zip file lol and havent spotted the issue yet to fix it lol.
Also bjgood you should be able to post now if ya see this.

And for a quick update to the scrubs addon changes, im still about half way done but distracted by a new video game so coding is on the back burner lol

**Comment by bjgood** [comment]
Thanks, JewBMX. I really appreciate this, bud. :).
Later...

**Comment by jewbmx** [comment]
If anyones using keybase and has loot in the wallet that they wanna move or play with more i suggest you look into anchor usd. Its got alot going that makes it good to use lol. Think the site is like tryanchor.com

**Comment by jewbmx** [comment]
Also for anyone waiting on a new version of scrubs, i seem to be at the end of my long journey so once i make some new scrapers and possibly resolvers for said scrapers, i should be ready to redo my repo and start sharing my little project again as a fully working plugin lol. Altho imma probably need to do a little testing and cleanup afterwards then wait for yall to point out any errors or issues i might still have 😀 or even some shit i might not of completed lol

**Comment by jewbmx** [comment]
went ahead and tossed up the new repo and scrubs if anyone here was waiting.

**Comment by bjgood** [comment]
Hey JewBMX,
Scrubsv2 is working fine in Leia, but, at least when I tried it on my RPi4, in Matrix, it didn't work. Thanks for all those extra sources, dude. I'm getting a lot of links when I do a search.
If you want me to try my RPi4 with Matrix again, with Scrubsv2, let me know.
Later...

**Comment by jewbmx** [comment]
I dont know the little gay nicknames they have for their versions like Android lol. But if your saying you have found a problem i might be able to fix it for ya lol

**Comment by jewbmx** [comment]
i just fixed a pretty big error where my foolish ass forgot to add in six.ensure_str(string) for most the base64 decode lines, it was causing a whole lot of info grabbing errors lol. also went ahead and removed alot of log_utils lines that were cluttering up my log which likely clogged other folks up too lol.

also as a side note, if anyone dislikes the new changelog loading on a new version feature, let me know and i will remove it or add a setting to disable it.  i kinda like it and hate it lmao.

**Comment by jewbmx** [comment]
I fixed a few more little issues and keep forgetting to update the changelog to point em out but o well lol. Also added that option to disable the changelog pop up.

**Comment by bjgood** [comment]
Scrubsv2-5.0.4 is working fine on my RPi4>LibreELEC 10.0.2>Matrix.
Thanks for the corrected version. :).
Later...

**Comment by jewbmx** [comment]
I fixed a couple more little things that me and a few other people spotted and fixed a couple scrapers i broke with a missing import last update lol. But waiting till later today/tonight to upload incase more stuff needs touched.

Oh and i still dont know your version jibberish lol. All i understood there was my addons current version is working fine lmao

**Comment by xalaros** [comment]
RD links don't appear.

**Comment by jewbmx** [comment]
I removed all debrid based code or atleast i think i got it all lol.
But kinda left the debrid hostDict to see if anyone using debrid has em show up.
I figured if your resolveurl is setup for it maybe they will still work for you. But i was still aiming for it all to be gone lol.
If they do show up it is probably normal color and just a "rare" host in the list. But if your all about debrid like many users id suggest using TheOath which is a great addon and more for debrid use. (I also share my code with him so he might have my scrapers added in randomly.)

**Comment by jewbmx** [comment]
Gonna be adding in a new indexer soon that has a pretty hefty amount of sites in it that are each dedicated to a show. Seems pointless to use em anywhere else so this will be a fun way to play with em lol. A bonus is that some have the special features of some shows and movies included. Eventually i will also do this for some anime and toon sites as well  but thats gonna be a bit different lol

**Comment by jewbmx** [comment]
Its very time consuming to put 70 sites/pages into one indexer lol

**Comment by jewbmx** [comment]
If anyone like the show The Pretender and wants to watch the movies, the second movie seems impossible to find so you might need to use a browser and find it on watchserieshd.stream hidden in season 4 as episode 22. Also episode 21 is the first movie but the first ones easily found lol.

**Comment by jewbmx** [comment]
If anybody uses the new UStvGO section and happens to try the Local category, id be happy to hear of your results or lack there of lol.
For me it works in a browser but fails in kodi and since its browser results still function properly when tossed in kodi im aiming to keep the category and find a way for it to work proper in kodi all the time.

**Comment by jewbmx** [comment]
Kinda wanna code a poll into scrubs for this lol but feelin lazy today.

Im thinking about ditching the quality style i currently use and wanna see what everyone else thinks...

Im thinking ditch all the 4k 1080p bullshit and going back to the old-fashioned style of HD or SD. Most the time quality is wrong or the host has other quality versions in the link so a sd can be 1080p and a 1080p can totally end up being a 480p. So why not say fuck it and revert back to a classic style thats a little more accurate and reduces my total file size and lines of code lol.

What yall think? (And yes i know i just spent all that time coding in the extra qualities and finally getting used to the scrape display lmao.)

**Comment by bjgood** [comment]
Hey JewBMX,
I think it's a good idea. I've noticed the quality variation as well. I never choose 1080p, nor 4k, anyways. 
Later...

**Comment by jewbmx** [comment]
I got myself stuck on another video game so it might be a little bit before i do another update lol but plan to change it anyways and possibly swap back if i dont end up liking it. Thanks to this game everyone should have a valid amount of time to put their opinion in tho before i do it lol.
My ideas probably HD and SD then im thinkin about adding something like this in the info bit to still show the quality found...
HD | Direct | (1080p?) | blah blah...
HD | Direct | (720p?) | blah blah..
SD | Direct | (480p?) | blah blah.

**Comment by jewbmx** [comment]
I could totally just wipe out all quality shit tho and be fine with just the scraper name and host name lol. I dont really care about any other details 🙂

**Comment by bjgood** [comment]
Hey JewBMX,
Do you know what's going on with host505? Looks like he's stopped development of The Oath. It can no longer be found on his github.
Later...

**Comment by jewbmx** [comment]
Be patient and it will likely be back around with some changes, or some random person will probably copy it and release it as their addon like usual lol.
Just reminds ya how fucked the world is and how money is the root of all evil lmao

**Comment by bjgood** [comment]
I'm not a religious person, but "amen" to what you said.

**Comment by jewbmx** [comment]
From what im hearing the only addons really falling ill lately are the ones that started using those official services like hulu. I wouldn't be surprised if the little cock smokers have a hunting party paid to make sure people like us dont cock block em from their bogus ass ad revenue lol

**Comment by jewbmx** [comment]
I love all the paid users that keep crying to me about their debrid service not working in my addon lmao. Apparently nobody  can read where i added that info into the addon to warn the prissy little fools. As if i give a fuck lmao

**Comment by AT5Pin** [comment]
LOL, nope, most don't read any part of what they're doing with this stuff and what makes it work. instead they want to be spoon fed and become offended if some other person doesn't read it for them. wah, fugg-n wah. I honestly believe we lost two good devs because if this bullshit they spew forth demanding help to fix their shit. jewbmx, you do you man! know you will and it's all good! thanks for that.

**Comment by jewbmx** [comment]
Lol i thought id be a bit obvious with the changelog notes and the lack of debird settings, but o well lol. The whole point of it is that i dont use the shit and dont wanna maintain it blind, plus i wanna force my user base to be free type of people only so them debird fuckers dont go wasting them scrapers away without ever needing em lol.
And i dont think we really lost any devs, they are likely still gonna do their thing but it might be in a more private fashion,  the addiction of coding is way to strong to quit cold turkey lol.

**Comment by AT5Pin** [comment]
If they don't want to invest a couple of minutes of their time to see what the Developer is doing with it in his provided notes; I'm not the brightest, but I can read. This old crab chuckles about that because it was obvious to me your intended direction.
So, I'm with ya, it's your do and you do you and it is all appreciated. 
I reckons as far as losing these devs, it is more about their continued work going away, for the time being. Sheeit man, I'm still missing tempest LOL... I really do hope that cat is doing well. Take cares man...

**Comment by jewbmx** [comment]
Lol to all that and i yea i havent seen or heard from tempest in forever.

To go off topic and save a post, imma add here that..
everyone that sends me emails should make the Subject : Scrubs  or Scrubsv2  that way its easy to find your emails.
Just spent like a half hour scrolling thru a huge spam folder digging folks out because their emails got auto'd lol. Imma pop the same little side note into my addon next time i update and might look into the email aspect of python to see if there can just be a lazy report issue option coded in that works with kodi. (Means every email would be from like jewbot1@gmail with everything hard coded besides the main post text aka body.)

**Comment by jewbmx** [comment]
Hmm just realized the abc site is the same as freeform lol. 
Decided to watch the 3 fallen movies since they are in the tmdb collections section, by the way those 2 sites are probably the best source for the movies if anyone decides to watch em and sees this lol

**Comment by bjgood** [comment]
Hi JewBMX,
How's it going? Any updates headed our way?
Have a good one. :).
Later...

**Comment by jewbmx** [comment]
How about a update that fixes nothing but pops up ok boxes every 5 minutes telling people that debrid bullshit wont work in scrubs lol

**Comment by bjgood** [comment]
> How about a update that fixes nothing but pops up ok boxes every 5 minutes telling people that debrid bullshit wont work in scrubs lol

Funny. :).

**Comment by jewbmx** [comment]
Just noticed my firestick was running slow and it turned out to be a user error lol, i forgot to use kcleaner to ditch all the artwork and addon packages that have been adding up since i bought the thing. Sad parts that ive had kcleaner installed since before my own addon lol

**Comment by jewbmx** [comment]
Looks like i need to update my code again in the scrapers and make a few changes mainly in what seems to be the ones using the cookie code stuff at the start like a session. Guessing if i make the init be try: except:   it will solve the delay in the scrape startup that is more noticeable with movies. Also might have to make a couple new ones in order to get some more movies and shows that my current pile dont have sources for.
Most likely gonna take the quality stuff old school like i previously discussed too while im at it since the debrid bullshit has finally settled down lol gotta give people something to complain about ya kno

**Comment by bjgood** [comment]
Hey dude,
An update will be welcome. No doubt about that. 
BTW, have you tried a grow tent yet? I have. They're good for saving money...if you know what I mean. ;).
Later...

**Comment by bjgood** [comment]
Hi,
Any timeline on the next update?
Thanks.
Later...

**Comment by jewbmx** [comment]
I see how it is, bug me repeatedly for an update then cant even say thanks when it comes 😢😥😢😥

**Comment by jewbmx** [comment]
if anyone else has decided to rewatch the old csi shows and/or is interested in the 2 part finale aka "immortality" i found a link for it here  https://wvv1.123movieshub.tc/movie/csi-immortality/watching.html?ep=1&sv=8

i might go ahead and try to fix this item in the addon so it will play proper instead of the ghetto behind the scenes bullshit episode that is currently the only sources found lol. might just be a easy custom labeling fix needed.
also might go ahead and try  to make a scraper from this site too if it isnt already one i have lol.

**Comment by bjgood** [comment]
> I see how it is, bug me repeatedly for an update then cant even say thanks when it comes crydisappointed_relievedcrydisappointed_relieved
Sorry for not replying. Thanks for the update, JewBMX. :).
Later...

**Comment by jewbmx** [comment]
Apologies early to everyone for the annoying ok dialog popup about the addon being a debrid free zone lol.
The good news is that it will only pop up once when you update aka the cache version check at the startup of the addon. The bad news is that i didnt code in any way for yall to disable that msg lol.

**Comment by jewbmx** [comment]
Heads up to anyone watching CSI Miami, season 3 episode 15 seems to only be in another language lol. I checked all the sources and they all were the same so you can save your time and skip passed it 😉 (didnt hunt down a english copy so no hookups lol)

**Comment by jewbmx** [comment]
I was just thinking that making the addons default click on a item goto information instead of play would be kinda useful and nice, but with just one test i already decided its not wise lol. When ya click play from information it seems to scrape then attempt to play the first link without letting ya pick one first lol. Shame it didnt go the same way my brain did

**Comment by jewbmx** [comment]
Im probably a long way off from being done with my current iptv project so nows a good time to ask yall this question lol giving plenty of time for various opinions..

Would you rather have a iptv menu thats sorted by site/provider or sorted by channel with a scrape style result showing all sites/providers that are available for like AMC lol.
One of my tv's or firestick has a combined listing similar to this idea and theres also some ghetto sites out there that do something like it too listing in a movie or show is on the various legit streaming services lol. (Same thing basically)

**Comment by bigpoppapaul** [comment]
> 

Personally, I'd definitely like the latter: channel with a scrape style result. That way you don't have to worry about  going from provider to provider until you find a working version of the channel you want to watch. When one's down, you just go to the next scraped selection.

**Comment by bjgood** [comment]
I agree with bigpoppapaul.

**Comment by jewbmx** [comment]
I think ive come up with the perfect way to do the whole thing lol now i just need to write some of the code and add in my notes before i forget any of it.

**Comment by jewbmx** [comment]
Dont know if anyone has used it yet but the collection search is dope as fuck lol made it alot easier to search up the naruto movies and have em sorted like i wanted 😀
Altho fair warning that i think i still need to code it for shows, if im not mistaken i only have it half way coded there.

**Comment by bjgood** [comment]
Hey JewBMX,
I don't know how you bypassed USTVGo's VPN block on their website, but you did (which I discovered yesterday). Congrats.
Later...

**Comment by jewbmx** [comment]
Cant really remember the code right now lol but i think my indexer script just scrapes the site from like page one onward or it might be coded with the channel links then scraped from that page.  Hard to say because ive done like 3 of em for that site lmao. Also not real sure how i coded it past that for the play code either lol but glad the geo block can suck it :)

**Comment by jewbmx** [comment]
While this stuffs on my mind still, you should know that the other iptv site i think below the option you picked is a twin site for that one your using so if one fails or is missing a channel you should be able to pop over to the other and get what ya want.
Oh and if you or anyone else is curious about that newer iptv project i started, im only about 1/3 of the way thru the list making and haven't even begun to write any code for it lmao. Its still a bunch of .m3u files and my hand made json files lol im totally slackin

**Comment by bjgood** [comment]
That VPN bypass I mentioned works on both ustv247 and USTVGo. I was highly surprised that your code worked. Great stuff.
Thanks for the reply, dude.
Later...

**Comment by jewbmx** [comment]
If anyone has trouble with finding a show in their watchlist that the addon doesnt load in due to its setup, like clicking progress but kung fu isnt one of the 40ish loaded in. I have found a kinda easy workaround, i swap to the menu item Episodes then scroll thru the pages and find the better shows that seem to get left behind lol. After that ya just watch a episode or season lol and the show pops back up in the normal progress section.
This is hopefully just a temporary quick fix tho because im planning to try and make the whole setup work quickly and actually load everything which means us big list users might load slowish still due to our list sizes but it should be as fast as possible without losing any shows in the process. 
(I finally got my show list below 100 active items too so i am eager to toy with the code before it pops back up lol.)

**Comment by jewbmx** [comment]
Looks like i finally found my issue with the progress list and i only had to ditch one line of code in my scrapePage def.
Might still try redoing the episodes indexer eventually but for now imma run the fuck away from it lmao.
Updates coming soon with my current changes but doing some light scraper updates first before someone asks me to do it anyways lol

**Comment by bjgood** [comment]
> Looks like i finally found my issue with the progress list and i only had to ditch one line of code in my scrapePage def. Might still try redoing the episodes indexer eventually but for now imma run the fuck away from it lmao. Updates coming soon with my current changes but doing some light scraper updates first before someone asks me to do it anyways lol

I have a sneaking suspicion you're referring to me, JewBMX. Ha!
Have a good one.
Later...

**Comment by jewbmx** [comment]
Lol your about 1 out of 5 people that would 😉
Think i was too high earlier tho because i recall doing a quick url test on each scraper to ditch duds but pretty sure i forgot to actually do test scrapes in the addon lol.

**Comment by jewbmx** [comment]
If anyone notices and wonders, i coded the newest setting in backwards and it will be fixed in the next update lol. It was my lazy way to ditch the shit and set the setting.
The codes already updated now and my setting is enabled on my end so now its just a light relief to other users lol and hopefully yall will spot the setting before your next update so it dont bug ya either 😉

**Comment by jewbmx** [comment]
Seems my last update to fix some trakt issues paid off on yet another problem lol. If your watchlist is bigger then 100 items it might still act up but now when you reach the end of a season you can goto bed without the show being lost in progress page load. Before on lots of occasions you would need to finish season 2 and start watching season 3 to make sure your show stayed in the listing but now its not fucking up anymore 😀

**Comment by jewbmx** [comment]
If anyone knows their way around the addon code and wants to help hunt down a error, im looking for the issue that causes some stuff to error out like Princess Mononoke (1997)
I feel like its something in the final info prep of the indexers or the sources module but i haven't been able to narrow it down yet lol

**Comment by jewbmx** [comment]
I removed my recent change to include debrid for hosts on the resolve lol. While testing new scrapers i noticed that it pops up debrid bullshit unless you disable the debrid services in resolveurl settings so its been changed back now lol.

Also closed this issue/post and opened a new one so we dont have to scroll around as much, this ones getting bulky.
